sw/airborne/firmwares/rotorcraft/autopilot.c

```diff
@@ -573,6 +573,40 @@ bool autopilot_guided_move_ned(float vx, float vy, float vz, float heading)
   return false;
 }
 
```

Comment:
Added the function 
`bool autopilot_guided_goto_ned_optional(float x, float y, float z, float heading, bool use_x, bool use_y, bool use_z, bool use_yaw)`
Would this be ok for you?